app/assets/javascripts/discourse/app/controllers/user-posts.js

```diff
@@ -1,11 +1,3 @@
-import Controller, { inject as controller } from "@ember/controller";
-import { observes } from "discourse-common/utils/decorators";
+import Controller from "@ember/controller";
 
-export default Controller.extend({
-  application: controller(),
-
-  @observes("model.canLoadMore")
-  _showFooter() {
-    this.set("application.showFooter", !this.get("model.canLoadMore"));
-  },
-});
+export default class UserPostsController extends Controller {}
```

Comment:
Do we need to keep the file in this case? Or do you do it just to have a named class ?

Reply:
Yeah I guess it could technically be dropped, but having a named class feels nicer to me 👌